Sources/Responses/EntryContainer.swift

```diff
@@ -65,22 +65,23 @@ public class EntryContainer<T: BoxModel>: BoxModel {
         totalCount = try BoxJSONDecoder.optionalDecode(json: json, forKey: "total_count")
         offset = try BoxJSONDecoder.optionalDecode(json: json, forKey: "offset")
         nextMarker = try BoxJSONDecoder.optionalDecode(json: json, forKey: "next_marker")
-        limit = try BoxJSONDecoder.optionalDecode(json: json, forKey: "limit")
         entries = try BoxJSONDecoder.decodeCollection(json: json, forKey: "entries")
 
+        // The `limit` field is inconsistent in the API
+        // Sometimes it could be an Int and another time it could be a String.
+        // So as a work around we should check both of these types and try to decode original value.
+        if let limitValue = json["limit"] {
+            limit = Int(String(describing: limitValue))
+        }
+        else {
+            limit = nil
+        }
+
         // The `nextStreamPosition` field is inconsistent in the API
         // Sometimes it could be an Int and another time it could be a String.
         // So as a work around we should check both of these types and try to decode original value.
```

Comment:
So this comment is no longer valid?

Reply:
It's still valid but I replaced code with simple one